src/controllers/webhooks.js

```diff
@@ -1,27 +1,20 @@
 const models = require('../models');
+const { incrementTeamScore, decrementTeamScore } = require('./team');
 
 const noteAttributesEnum = {
-  userID: 0,
-  teamID: 1,
+  userId: 0,
+  teamId: 1,
   teamName: 2,
 };
 
 function findById(orderNumber) {
-  return models.Orders.findOne({
+  return models.Order.findOne({
     where: {
       orderNumber,
     },
   });
 }
```

Comment:
If we replace the usage of `orderNumber` with `id` in our model, we can replace this with `models.Order.findByPk()`
https://sequelize.org/master/manual/model-querying-finders.html#-code-findbypk--code-